booru-switcher.user.js

```diff
@@ -34,6 +36,7 @@ const boorus = [
   {name: 'Twibooru', host: 'twibooru.org', filterId: 2, bor: true},  // runs on Booru-on-Rails
   {name: 'Derpibooru', host: 'derpibooru.org', filterId: 56027},
   {name: 'Trixiebooru', host: 'trixiebooru.org', filterId: 56027},
+  {name: 'Manebooru', host: 'manebooru.org', filterId: 56027},
```

Comment:
To answer your question: `filterId` should be the ID of the filter with everything unhidden/unspoilered.

On Derpibooru, that filter is called '[Everything](https://derpibooru.org/filters/56027)' and as you can see from the url, its ID is `56027`.

The reason the filter is needed is because the sites would otherwise apply the default filter to any search queries, which on most sites will have explicit contents hidden, thus causing searches for those filtered images to fail.

Reply:
okay, so the filterID should be 2. Got it!